tests/async/test_async_queryset.py

```diff
@@ -243,6 +243,6 @@ async def test_aexplain(self):
                         self.fail(f"QuerySet.aexplain() result is not valid JSON: {e}")
 
     async def test_raw(self):
-        sql = "SELECT id, field FROM async_queryset_simplemodel WHERE created=%s"
```

Comment:
Removing `async_queryset` app is unrelated cleanup. Please move it to a separate commit/PR.